packages/draw/src/mindgraph.ts

```diff
@@ -0,0 +1,73 @@
+import { GraphData } from '@mindgraph/types';
+import { Artist } from './artist';
+import { Drawable } from './canvas';
+import { RenderableLink, RenderableNode } from './renderables';
+import { Simulation } from './simulation';
+import { GraphSimulationConfig, SimulationNode } from './types';
+import { Styles } from './style';
+
+export type NodeClickCallback = (node: SimulationNode) => void;
+
+export class MindGraph {
+  private simulation: Simulation;
+  private artist: Artist;
+  private callback: NodeClickCallback | undefined;
```

Comment:
nit: in this project I have been putting private properties and methods under public methods and properties.

Reply:
Oh I see that now, thanks!